### PR TITLE
Update README.md for first OSS release

### PR DESCRIPTION
**Pull request by billnitzberg**
No JIRA Ticket -- this is a simple text change to the README.md file.

### CODE REVIEW

**Comment by subhasisb** [comment]
Looks good. I sign off

File: README.md [hunk 1]
```diff
@@ -14,31 +12,31 @@ PBS Professional® software optimizes job scheduling and workload management in
...
+The best place to start is by joining the community forum.  You may sign up or view the archives via:
 
-* announce@pbspro.org -- important updates relevant to the entire PBS Pro community
-* users@pbspro.org -- general questions and discussions among end users (system admins, engineers, scientists)
-* dev@pbspro.org -- technical discussions among developers
+* [Announcements](http://community.pbspro.org/c/announcements) -- important updates relevant to the entire PBS Pro community
+* [Users/Site Admins](http://community.pbspro.org/c/users-site-administrators) -- general questions and discussions among end users (system admins, engineers, scientists)
+* [Developers](http://community.pbspro.org/c/developers) -- technical discussions among developers
 
 To dive in deeper and learn more about the project and what the community is up, visit:
```
Comment (subhasisb) [inline README.md]: Should probably read "what the community is up to" - missing the to?

File: README.md [hunk 2]
```diff
@@ -14,31 +12,31 @@ PBS Professional® software optimizes job scheduling and workload management in
...
 
 * [OpenHPC.community](http://openhpc.community)
 * [The Linux Foundation](http://thelinuxfoundation.org)
 
-## Our Vision:  One Scheduler for the whole HPC World
+### Our Vision:  One Scheduler for the whole HPC World
 
 There is a huge opportunity to advance the state of the art in HPC scheduling by brining the whole HPC world together, marrying public sector innovations with private sector enterprise know-how, and retargeting the effort wasted re-implementing the same old capabilities again and again towards pushing the outside of the envelope.  At the heart of this vision is fostering common standards (at least defacto standards like common software).  To this end, Altair has made a big investment by releasing PBS Pro under an Open Source license (to meet the needs of the public sector), while also continuing to offer PBS Pro under a commercial license (to meet the needs of the private sector).  One defacto standard that can work for the whole HPC community.  See Bill’s Open Letter to the HPC Community for more details.
```
Comment (subhasisb) [inline README.md]: bringing spelling is wrong

**Comment by subhasisb** [comment]
Looks good now. Thanks - i sign off

**Comment by billnitzberg** [comment]
Thanks a lot!  I fixed both.

**Pull request merged**